services/community-site/src/pages/Delegation/Delegation.tsx

```diff
@@ -434,7 +447,7 @@ const Delegation = ({ location }: { location: Location }) => {
                           )?.rewards || BigNumber.from('0')
                         }
                         currentBlockNumber={currentBlock}
-                        actionsDisabled={status !== 'connected' || !account}
+                        actionsDisabled={status !== 'connected' || !account || !user}
```

Comment:
We should use `isNotLoggedOrKycEd` here too

Reply:
Sure, adding it now